ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_dialog/.content.xml

```diff
@@ -106,7 +106,7 @@
                             </enums>
                             <bindref
                                     jcr:primaryType="nt:unstructured"
-                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    sling:resourceType="fd/af/authoring/components/granite/form/bindrefselector"
```

Comment:
Is this part of public region, else content classification tool would throw an error

Reply:
Already using this resource for dorBindRef at line #116 .